packages/modal-ui-js/src/lib/components/WalletNotInstalled.ts

```diff
@@ -0,0 +1,34 @@
+import { ModuleState, Wallet } from "@near-wallet-selector/core";
+import { modalState } from "../modal";
+
+export function renderWalletNotInstalled(module: ModuleState<Wallet>) {
+  if (!modalState) {
+    return;
+  }
+
+  document.querySelector(".modal-right")!.innerHTML = `
+    <div class="nws-modal-body">
```

Comment:
Can we please align this component and make it the same as the react modal in the redesign PR?

To see the difference please run the react example and check an Uninstalled injected wallet. Also, the blue "Open Wallet Name"  button should redirect to the extension.